### DISCUSSION TITLE
logs not displayed in hunt

### DISCUSSION THREAD

**Discussion by securityfai** [Security-Onion-Solutions/securityonion#12812] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

We noticed that sometimes we are not able to see logs in our HUNT and KIBANA interfaces within SOC until we reboot our search nodes and the manager, What could be the problem?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
There could be any number of causes for this. Is there anything interesting in the logstash or elasticsearch logs on any of the nodes? They're found under /opt/so/log/logstash and /opt/so/log/elasticsearch, respectively.
